package.json

```diff
@@ -6,15 +6,19 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.25",
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
+    "history": "^4.10.1",
     "node-sass": "^4.13.0",
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
     "react-magic-slider-dots": "^1.1.3",
     "react-mailchimp-subscribe": "^2.1.0",
+    "react-redux": "^7.1.1",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
     "react-slick": "^0.25.2",
     "react-typing-animation": "^1.6.2",
     "react-typing-effect": "^2.0.2",
+    "redux": "^4.0.4",
     "slick-carousel": "^1.8.1"
```

Comment:
@Rebecasa check package versions with `npm list depth=0` and make versions fixed in `package.json` to prevent accidental changes and crashes.
example:
`"history": "^4.10.1",` => `"history": "4.10.1",`